### ISSUE TITLE
It takes a long time to fine-tune MiniCPM2.5v.

### ISSUE THREAD

**Issue by XuRui314** [modelscope/swift#1108]
I followed
https://github.com/modelscope/swift/blob/283b735a8e6b5f21d2f57da4c9b73e4ebf7c80f3/docs/source/Multi-Modal/minicpm-v-2.5%E6%9C%80%E4%BD%B3%E5%AE%9E%E8%B7%B5.md

This is my training log:
Train:   0%|                                                                                                                                                             | 2/2582 [02:08<46:11:57, 64.46s/it]

But finetuning GLM4 in the same setting only takes 9h, could you provide some advice for solving this problem?

**Comment by XuRui314** [comment]
One more problem about training loss. I used the official training script provided by MiniCPM-V. The loss converged to 1.0 while 1.5 using swift in the same training setting. Both the finetuned models underperformed in the test set. I think this is due to the insufficient loss decrease.
 
I also encountered this problem when tuning GLM4: 
https://github.com/modelscope/swift/issues/1091#issuecomment-2157788336
